src/server/blockchain/AdminWallet.js

```diff
@@ -170,7 +170,7 @@ export class Wallet {
     const adminWalletContractBalance = await this.web3.eth.getBalance(adminWalletAddress)
     log.info(`AdminWallet contract balance`, { adminWalletContractBalance, adminWalletAddress })
     if (adminWalletContractBalance < adminMinBalance * this.addresses.length) {
-      log.error('AdminWallet contract low funds', {})
+      log.error('AdminWallet contract low funds')
```

Comment:
@yaroslav-fedyshyn-nordwhale probably we may generate an exception and log it ?

```javascript
const adminWalletBalanceRequired = adminMinBalance * this.addresses.length

if (adminWalletContractBalance < adminWalletBalanceRequired) {
  const exception = new Error('AdminWallet contract low funds')
  const { message } = exception

  log.error('Error initializing wallet:', message, exception, { adminWalletContractBalance, adminWalletBalanceRequired })
  if (conf.env !== 'test' && conf.env !== 'development') process.exit(-1)
}

Reply:
@sirpy could you explain please why process.exit() ? could we just throw an exception to break initialization process ?